### PR TITLE
G-Mode Aqueduct, Crab Shaft

### PR DESCRIPTION
**Pull request by kjbranch**
I went back and forth on ways to bypass snail climbs, and eventually settled on ignoring them. It ended up making way too many things far too complex. 

I doubt it would ever come up that a player can do these g-mode strats without the snail climb, but could be a problem in an enemizer (along with many other things)

### CODE REVIEW

File: region/maridia/inner-pink.json [hunk 1]
```diff
@@ -1004,6 +1165,143 @@
...
+                      {"itemNotCollectedAtNode": 7},
+                      {"itemNotCollectedAtNode": 8}
+                    ]},
+                    "SpringBall",
+                    {"ammo": { "type": "PowerBomb", "count": 2 }},
+                    "h_canNavigateUnderwater",
+                    "canSnailClimb",
+                    {"or":[
+                      "Gravity",
+                      "h_canMaxHeightSpringBallJump",
```
Comment (osse101) [inline region/maridia/inner-pink.json:1288]:
```suggestion
```

Comment (osse101) [reply region/maridia/inner-pink.json:1288]: With morph you take a different strat and that makes this option just confusing.

File: region/maridia/inner-pink.json [hunk 2]
```diff
@@ -716,6 +737,30 @@
...
+                    }},
+                    "Gravity",
+                    "h_canArtificialMorphIBJ",
+                    {"or": [
+                      "h_canArtificialMorphCeilingBombJump",
+                      "Morph",
+                      "SpringBall"
+                    ]}
+                  ],
+                  "note": "Bomb the crumble blocks above to overload PLMs, then IBJ to escape. IBJ through the Speed Blocks to get to the items."
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-pink.json:760]: You can't overload PLMs by bombing PB or Super blocks

File: region/maridia/inner-pink.json [hunk 3]
```diff
@@ -872,6 +936,103 @@
...
+                {
+                  "name": "G-Mode Overload PLMs - Bomb the Speed Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,5,6],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }},
+                    "h_canUseMorphBombs",
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-pink.json:976]: For all the hours I was planning on getting through these bomb blocks, I should have thought of using bombs on them .......

File: region/maridia/inner-pink.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (osse101) [reply region/maridia/inner-pink.json:760]: Sure you can, when it reveals the block.

File: region/maridia/inner-pink.json [hunk 4]
```diff
@@ -1226,6 +1502,43 @@
...
+                          "HiJump",
+                          "canUseEnemies"
+                        ]},
+                        {"and": [
+                          "Bombs",
+                          "HiJump",
+                          "SpringBall",
+                          "canSnailClimb"
+                        ]}
+                      ]}
```
Comment (osse101) [inline region/maridia/inner-pink.json:1536]:
```suggestion
                        ]}
                      ]},
                      {"and": [
                        "Bombs",
                        "HiJump",
                        "SpringBall",
                        "canSnailClimb"
                      ]}
```

**Pull request merged**